src/GameServer/Bot/Population/PopulationService.js

```
@@ -32,6 +32,14 @@ const PlayerActivitySignal = invoke('GameServer/Bot/Population/PlayerActivitySig
 const FloorAwareActivationPolicy = invoke('GameServer/Bot/Population/FloorAwareActivationPolicy');
 const ColdSimulationOwner = invoke('GameServer/Bot/Population/ColdSimulationOwner');
 const ColdSimulationCoordinator = invoke('GameServer/Bot/Population/ColdSimulationCoordinator');
+const PartyRequestPlanner = invoke('GameServer/Bot/Population/PartyRequestPlanner');
+const BackgroundPartyLifecycle = invoke('GameServer/Bot/Population/BackgroundPartyLifecycle');
+
+const {
+    partyObjectiveForPlan,
```

ESLint

[error] 'partyObjectiveForPlan' is assigned a value but never used.

(no-unused-vars)